### DISCUSSION TITLE
How to write multiple blacklists in .config

### DISCUSSION THREAD

**Discussion by jp1151419** [UltimaHoarder/UltimaScraper#1431] [Q&A] [answered]
I'm pretty sure I read in a comment at one point you can have more than one blacklist in the settings but I can't seem to make it work. 

I tried these two ways and both caused an error : 
"blacklist_name": "blacklist,blacklist2",
"blacklist_name": "blacklist","blacklist2",

**Comment by UltimaHoarder** [comment]
Try converting it to a list with the blacklist names.

I think I may have it hardcoded to support one though.

**Comment by cjemorton** [comment]
Woops! I just noticed this thread after posting a question. To Issues.
https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1476 
I have the same question.

**Comment by UltimaHoarder** [comment, marked as answer]
It's been added, but as a list
